src/components/input-field/input-field.scss

```diff
@@ -105,9 +105,15 @@
     + .mdc-text-field__input {
         z-index: z-index.$input-field--input-with-formatted-value;
     }
+}
+
+.lime-has-prefix:not(.mdc-text-field--focused):not(.mdc-text-field--invalid) {
+    .lime-formatted-input {
+        position: unset;
+    }
```

Comment:
We think it could be better to apply this styling instead of setting the 
`position: unset;` Remove lines I have commented out 
(line 92) :

```scss
.lime-formatted-input {
    display: none; // Add this one
    width: calc(100% - 1.25rem);
    z-index: z-index.$input-field--formatted-value;

    // position: absolute;
    // top: 0.9rem;
    pointer-events: none;

    // opacity: 0;
    :not(.mdc-text-field--focused):not(.mdc-text-field--invalid) & {
        // opacity: 1;
        display: block; // Add this one
    }

    + .mdc-text-field__input {
        z-index: z-index.$input-field--input-with-formatted-value;
    }
}

//.lime-has-prefix:not(.mdc-text-field--focused):not(.mdc-text-field--invalid) {
//.lime-formatted-input {
// position: unset;
// width: calc(100% - 1.25rem);
//}

// .mdc-text-field__input[type='number'] {
//     width: 0;
// }
//}

```